security.rst

```diff
@@ -2117,7 +2117,7 @@ Prepending the path with ``^`` means that only URLs *beginning* with the
 pattern are matched. For example, a path of ``/admin`` (without the ``^``)
 would match ``/admin/foo`` but would also match URLs like ``/foo/admin``.
 
-Each ``access_control`` can also match on IP address, hostname and HTTP methods.
+Each ``access_control`` can also match on IP address, hostname and HTTP methods or be directly configured by passing a service implementing `RequestMatcherInterface` through the `request_matcher` option.
```

Comment:
I would split this into two sentences to make this piece of information easier to digest.

Reply:
done :)